console/client/src/components/CodeBlock.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react'
-import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
-import { atomDark, oneLight } from 'react-syntax-highlighter/dist/esm/styles/prism'
-import { useDarkMode } from '../providers/dark-mode-provider'
+import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter'
+import {
+  atomDark,
+  oneLight,
+} from 'react-syntax-highlighter/dist/esm/styles/prism'
+import {useDarkMode} from '../providers/dark-mode-provider'
 
 type Props = {
   code: string
   language: string
 }
 
-export const CodeBlock: React.FC<Props> = ({ code, language }) => {
-  const { isDarkMode } = useDarkMode()
+export const CodeBlock: React.FC<Props> = ({code, language}) => {
```

Comment:
Another example where the linter could catch that we're not specifying type here.

The other PR changed this code to this, which feels a bit more explicit.
```ts
interface Props {
  code: string
  language: string
}

export const CodeBlock = ({ code, language }: Props) => {
```